Test/PacketType/L2tpPacketTest.cs

```diff
@@ -34,5 +35,24 @@ public void L2tpParsing()
             Assert.AreEqual(l2tp.SessionID, 54110);
             Console.WriteLine(l2tp.GetType());
         }
+
+
+        // L2TP
+        [Test]
+        public void L2tp_SessionID()
+        {
+            var p = new L2tpPacket(new PacketDotNet.Utils.ByteArraySegment(new byte[8]
```

Comment:
Here is a wireshark of the example packet I am using, I just copied over the bytes for ease of testsing

![image](https://user-images.githubusercontent.com/50456180/102627056-a4353080-4115-11eb-9b0a-ae2929165461.png)